apps/portlet-documents/src/main/webapp/js/attachmentService.js

```diff
@@ -271,7 +266,34 @@ export function getAttachmentById(attachmentId) {
     if (resp || resp.ok) {
       return resp.json();
     } else {
-      throw new Error('Error getting attachments task');
+      throw new Error('Error getting entity\'s linked attachment');
+    }
+  });
+}
+
+export function moveAttachmentToNewPath(newPathDrive, newPath, attachmentId, entityType, entityId) {
+  if (!attachmentId) {
+    throw new Error('Attachment Id can\'t be empty');
+  }
+  let params = {};
+  params.newPath = newPath? newPath: '';
+  if (newPathDrive) {
+    params.newPathDrive = newPathDrive;
+  }
+  if (entityType) {
+    params.entityType = entityType;
+  }
+  if (entityId) {
+    params.entityId = entityId;
+  }
+  params = $.param(params, true);
+
+  return fetch(`${eXo.env.portal.context}/${eXo.env.portal.rest}/attachments/${attachmentId}/move?${params}`, {
```

Comment:
It would be better to pass the parameters in Body instead of query (Possible Seurity breach, where the names are in the URL and thus not encrypted with TLS/SSL)